src/translations/en.json

```diff
@@ -397,13 +397,8 @@
     "sort": {
         "duration": "Duration",
         "position": "Position",
-        "position DESC": "Position (descending)",
```

Comment:
How is it possible that this reverted ?
this should be position_desc as key in the language file.

are we 100% sure that lokalize is not overwriting our base file ?

Reply:
Some corrections I made to the en base got reverted by Lokalise automatically I think. ISTR that there's some setting in Lokalise to allow changing/deleting existing entries.

Reply:
Not sure. I wonder if it happened because I accidentally changed the English file…although it should have had the latest..? I will keep a close eye on things once this goes through.